### PR TITLE
clases.html

### PR DESCRIPTION
**Pull request by ioandodita069**
this is my second lesson, the first one it's already merged..sorry, it wont happen again

### CODE REVIEW

File: html/clases.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    <style>
+        .city {
+          background-color: tomato;
+          color: white;
+          padding: 10px;
+        }
+        </style>
+        </head>
+        <body>
```
Comment (pancudaniel7) [inline html/clases.html:50]: You have multiple <body> task, in one html file you should have jsut one <body> one <head> and one <html> tag

File: html/clases.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+<h2>Paris</h2>
+<p>Paris is the capital of France.</p>
+</div>
+
+<div class="city">
+<h2>Tokyo</h2>
+<p>Tokyo is the capital of Japan.</p>
+</div>
+
+<style>
```
Comment (pancudaniel7) [inline html/clases.html:30]: Aslo the <style> block should be added just inside <head> block, in your cases you should have just one <style> added inside of <head> tath starts at line 2

**Review by pancudaniel7 — COMMENTED** [review]
If you don't understand my comments lets have a talk

**Comment by pancudaniel7** [comment]
Also on the title you can add directly the name of the 23schools lesson, for example you can add title lesson **Favicon** or **Tables** not **lesson2**

**Comment by ioandodita069** [comment]
i understand them very well

**Comment by pancudaniel7** [comment]
Ok if understand please make the changes and push again for me to verify

**Pull request closed without merging**